py_for_mxnet/initializer.py

```diff
@@ -0,0 +1,766 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Why do we need to hard copy all the initializers already existing in MXNet

Reply:
add the initalize case with params which end with "_minmax". This for aux_params in quantization int8 op.